src/components/JUPTable/JUPTable.tsx

```diff
@@ -33,17 +33,23 @@ export interface IHeadCellProps {
 
 interface ITableTitleProps {
   title: string;
-  path: string;
+  path?: string;
   DisplayedComponents?: Array<React.ReactElement>;
 }
 
 const TableTitle: React.FC<ITableTitleProps> = ({ title, path, DisplayedComponents }) => {
   return (
     <>
-      <TitleText variant="h6" id="tableTitle">
-        <SLink href={path}>{title}</SLink>
-        {DisplayedComponents}
-      </TitleText>
+      {path ? (
+        <TitleText variant="h6" id="tableTitle">
+          <SLink href={path}>{title}</SLink>
```

Comment:
why not just wrap the SLINK with the IF statement here? why the double titletext ?

Reply:
fixed